app/controllers/classified_listings_controller.rb

```diff
@@ -13,7 +13,9 @@ def index
     set_surrogate_key_header "classified-listings-#{params[:category]}"
   end
 
-  def show; end
+  def show
+    @classified_listing = ClassifiedListing.find_by(category: params[:category], slug: params[:slug])
```

Comment:
should this be `find_by!` ? in case the combination category and slug doesn't exist...